crates/moon/tests/test_cases/mod.rs

```diff
@@ -3858,7 +3860,7 @@ fn test_multi_process() {
 
                 if output.status.success() {
                     success.fetch_add(1, std::sync::atomic::Ordering::SeqCst);
-                    let out = String::from_utf8(output.stdout).unwrap();
+                    let out = String::from_utf8(output.stderr).unwrap();
```

Comment:
should this change to stderr?

Reply:
Yes, see below: the "no work to do/now up to date" line now resides in stderr instead of stdout (to not interfere with the actual run output), so it needs be changed here.